src/api/cpp/cvc5_kind.h

```diff
@@ -1868,7 +1868,7 @@ enum CVC5_EXPORT Kind : int32_t
    *
    * Note: We currently support the creation of constant arrays, but under some
    * conditions when there is a chain of equalities connecting two constant
-   * arrays, the solver doesn't know what to do and aborts (Issue <a href="https://github.com/CVC4/CVC4/issues/1667">#1667</a>).
+   * arrays, the solver doesn't know what to do and aborts (Issue <a href="https://github.com/cvc5/cvc5/issues/1667">#1667</a>).
```

Comment:
I'd suggest to avoid HTML and use Sphinx here:
```suggestion
   * arrays, the solver doesn't know what to do and aborts (Issue `#1667 <https://github.com/cvc5/cvc5/issues/1667>`_).
```
(Please check that is is really correct...)

Reply:
I compiled the docs and it seems to have worked.